Python/other/TripletSumValue.py

```diff
@@ -0,0 +1,58 @@
+'''
+Given an array arr and an integer k.Find if there's a triplet in the array which sums up to the given integer k
+Input:
```

Comment:
Need not write Sample IO at the beginning of the file. It has to be there only once and that too after the code.

Reply:
okay I will not write

Reply:
@bhav09, please check my modify file